### DISCUSSION TITLE
GitHub Pull Request Fails

### DISCUSSION THREAD

**Discussion by chimosky** [WeblateOrg/weblate#9810] [Q&A]
I'm running a self hosted weblate instance and I have configured [`GITHUB_CREDENTIALS`](https://docs.weblate.org/en/weblate-4.18.2/admin/config.html#std-setting-GITHUB_CREDENTIALS)  and it's selected as the version control for projects but fails with error `Pull request failed: Not Found `, error can be seen in the repository changes section of the manage page for the component.

There are no other logs that provided any helpful information.

According to https://github.com/WeblateOrg/weblate/discussions/6765 weblate fails to create the pull request and I'm guessing this hasn't been fixed.

**Comment by chimosky** [comment]
@nijel your help would be appreciated.

**Comment by nijel** [comment]
Weblate 5.0.1 will do a bit more verbose logging here. You can also enable Sentry integration to get more insights on errors like this in Sentry.

**Reply by chimosky** [reply]
I'll wait for 5.0.1, thanks!
